src/endpoint/s3/s3_utils.js

```diff
@@ -785,6 +785,15 @@ function key_marker_to_cont_tok(key_marker, objects_arr, is_truncated) {
     return Buffer.from(j).toString('base64');
 }
 
+function invalid_schema_to_aws_error(error) {
```

Comment:
Please add JSDoc to the new function with a short explanation of why/when to use it.

Reply:
Added, Thanks